scripts/aspenmesh-collector-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+if ! [ -x "$(command -v helm)" ]; then
+  echo "helm is not installed. Please visit https://helm.sh/docs/intro/install/" >&2
+  exit 1
+fi
+
+usage() {
+  echo "
+usage: ${0} [flags] [-h|--help]
+Flags               Purpose
+--apiKey            Required; The FusionAuth API key for the user's group
+--endpoint          Optional; The Promscale remote write endpoint. Default value: https://metrics.cloud.aspenmesh.io/write
```

Comment:
Let's remove this, as it's not necessary for end users. We only have one endpoint today, which the chart defaults to.

Reply:
Let's definitely remove it from the help, but keep the functionality it for internal use

Reply:
@tartale Rick also left a comment in the story and explained why one AC was to create a new `cli` repo: https://aspenmesh.atlassian.net/browse/AM-4659?focusedCommentId=13091. Thoughts?

Reply:
That seems fine to me. The ability to override `--endpoint` is important to me though.

I have a suggestion for that below that makes it less obvious that you can override; see below